pkg/trace/api/api_test.go

```diff
@@ -368,6 +368,42 @@ func TestReceiverDecodingError(t *testing.T) {
 	})
 }
 
+func TestReceiveEndpointSniffingRequests(t *testing.T) {
+	assert := assert.New(t)
+	conf := newTestReceiverConfig()
+	r := newTestReceiverFromConfig(conf)
+	server := httptest.NewServer(http.HandlerFunc(r.handleWithVersion(v05, r.handleTraces)))
+	client := &http.Client{}
+
+	t.Run("v0.5 detection request", func(t *testing.T) {
```

Comment:
On the same note as [this comment](https://github.com/DataDog/datadog-agent/pull/6326/files#r484391598), I think we can name these tests `v0.4` and `v0.5`